### DISCUSSION TITLE
Conditional Robotoff Questions

### DISCUSSION THREAD

**Discussion by teolemon** [openfoodfacts/robotoff#1208] [Ideas]
### What
- Derived from my old Robotoff+ proposal
- Generate Questions on packagings for as many products as possible
- Possibly extend that to origins using OCR triggers
```
trigger:category: en:eggs
fr:Cette boîte est-elle en carton ?
answer:type:yesno
answer:yes:answer_execution: packaging: fr:boite en carton
answer:no:answer_execution:

trigger:category: en:wines
fr:Le contenant est-il en verre ?
answer:type:yesno
answer:yes:answer_execution: packaging: fr:bouteille en verre
answer:no:answer_execution:


```
### Why
- We need more packagings, quickly

### Examples for categories
confitures (glass jar, metal lid), eggs (plastic box or cardboard box), juices (plastic bottle, plastic cap, glass bottle, metal cap)
### Part of
- #376

**Comment by mithridatea** [comment]
This feature is actually a subset of https://github.com/openfoodfacts/robotoff/issues/307 focused on packaging.

**Comment by teolemon** [comment]
yes indeed.
